src/system/applications/dialogs/attack-configuration.ts

```diff
@@ -207,7 +213,7 @@ export class AttackConfigurationDialog extends ComponentHandlebarsApplicationMix
         this.data.skillTest.temporaryModifiers = tempMod;
 
         const skill = this.data.skillTest.data.skill;
-        const attributeData = !isNone(attribute)
+        const attributeData = !isNull(attribute)
```

Comment:
Do we want to keep the `isNull` check in? One of the advantages of using `null` over `'none'` is that it's a falsy value

Reply:
True, I just didn't want to keep typing out the check long hand, so renamed the existing function once 😂
I can go back and swap it out next session I sit down at VScode